heft-plugins/heft-sass-plugin/src/SassTypingsPlugin.ts

```diff
@@ -1,21 +1,29 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
-import { IHeftPlugin } from '../../pluginFramework/IHeftPlugin';
-import { HeftSession } from '../../pluginFramework/HeftSession';
-import { HeftConfiguration } from '../../configuration/HeftConfiguration';
-import { IBuildStageContext, IPreCompileSubstage } from '../../stages/BuildStage';
+import * as path from 'path';
+import {
+  HeftConfiguration,
+  HeftSession,
+  IBuildStageContext,
+  IHeftPlugin,
+  IPreCompileSubstage,
+  ScopedLogger
+} from '@rushstack/heft';
+import { ConfigurationFile, PathResolutionMethod } from '@rushstack/heft-config-file';
+import { JsonSchema } from '@rushstack/node-core-library';
 import { ISassConfiguration, SassTypingsGenerator } from './SassTypingsGenerator';
-import { CoreConfigFiles } from '../../utilities/CoreConfigFiles';
-import { ScopedLogger } from '../../pluginFramework/logging/ScopedLogger';
-import { Async } from '../../utilities/Async';
+import { Async } from './utilities/Async';
 
 export interface ISassConfigurationJson extends ISassConfiguration {}
 
 const PLUGIN_NAME: string = 'SassTypingsPlugin';
+const PLUGIN_SCHEMA_PATH: string = path.resolve(__dirname, 'schemas', 'heft-sass-plugin.schema.json');
```

Comment:
```suggestion
const PLUGIN_SCHEMA_PATH: string = `${__dirname}/schemas/heft-sass-plugin.schema.json`;
```

This is faster than using `path`.

Reply:
I'm a bit cautious about this due to os compatibility. And as far as I can see `path.join()` and `path.resolve()` are used consistently throughout the repo.

Reply:
Yeah we're trying to move away from that. It's more expensive than you'd think. Slashes work fine on Windows.

Reply:
Ok, good to know!